src/main/java/me/zeroeightsix/kami/gui/kami/KamiGUI.java

```diff
@@ -247,7 +247,7 @@ public void onScroll(MouseScrollEvent event) {
         frame = new Frame(getTheme(), new Stretcherlayout(1), "Inventory Viewer");
         frame.setCloseable(false);
         frame.setPinnable(true);
-        frame.setPinned(true);
+        frame.setPinned(false);
```

Comment:
There's no need to make this change, it's unrelated to the pull and already done in master